api/lib/infrastructure/repositories/user-active-for-admin-repository.js

```diff
@@ -0,0 +1,22 @@
+const { knex } = require('../../../db/knex-database-connection');
+const OrganizationForUserAdmin = require('../../domain/read-models/OrganizationForUserAdmin');
+
```

Comment:
Question : pourquoi avoir créé un nouveau repo et ne pas avoir utilisé le `api/lib/infrastructure/repositories/organization-for-admin-repository.js` existant ?

Reply:
Autre remarque, je ne crois pas qu'on fasse des repos liés spécifiquement à un read model. @ajoanny tu as un avis là dessus ? Faudrait-il créer un repo associé au read model `OrganizationsForAdmin` ou bien le repo `OrganizationForAdminRepository` peut-être utilisé. On va retourner peut être plein de données en plus dans le second cas.